com/uber/kafkaSpraynozzle/stats/StatsDReporter.java

```diff
@@ -0,0 +1,61 @@
+package com.uber.kafkaSpraynozzle.stats;
+
+import com.timgroup.statsd.NonBlockingStatsDClient;
+import com.timgroup.statsd.StatsDClient;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Statsd StatsReporter implementation
+ *
+ * To initialize this class you need to provide it the path to a .properties file containing:
+ * statsd.host - the statsd host
+ * statsd.port - the statsd port
+ * statsd.statsPrefix (optional) - a prefix for all stat names
+ */
+public class StatsdReporter implements StatsReporter {
+    private final StatsDClient statsd;
+    private final String statsPrefix;
+    private final String statsdHost;
+    private final Integer statsdPort;
+
+    public StatsdReporter(String propertyFilePath) throws IOException {
+        Properties statsProperties = new Properties();
+        InputStream fileInputStream;
+        try {
+            fileInputStream = new FileInputStream(propertyFilePath);
+            statsProperties.load(fileInputStream);
+            fileInputStream.close();
+        } catch (FileNotFoundException e) {
+            System.out.println("Failed to find property file: " + propertyFilePath);
+            throw e;
```

Comment:
Re-throwing the exceptions will affect the stability of spraynozzle. There is no exception handling in the main class. I s'pose this sort of catastrophic behavior is alright since it would be clear to the user immediately that it can't start up.